### PR TITLE
I18n support

### PR DESCRIPTION
**Pull request by cap10morgan**
We need to pass a `?language=all` query param when getting state voter reg. methods via the API, and then voter-registration-works needs to forward that param to election-authority-works so that we get all the translations of the instructions (instead of just English).

Part of this [Pivotal card](https://www.pivotaltracker.com/story/show/114359809)

I also fixed a bug where we weren't specifically asking for the registration authority in the `voter-reg-authority` fn.

### CODE REVIEW

File: src/voter_registration_works/states.clj [hunk 1]
```diff
@@ -71,14 +76,15 @@
 
 (defn voter-reg-authority
   [state voter]
```
Comment (sduckett) [inline src/voter_registration_works/states.clj]: Is this app-state?

File: src/voter_registration_works/states.clj [hunk 2]
```diff
@@ -30,15 +34,16 @@
...
   (let [ocd-id (str "ocd-division/country:us/state:"
                     (-> state name str/lower-case))]
-    (election-authority ocd-id)))
+    (election-authority {:ocd-id ocd-id, :language language})))
 
 (defn registration-methods
-  "Returns the voter reg. methods for `state` from the state election authority."
-  [state]
-  (let [authority (state-authority state)
+  "Returns the voter reg. methods for `query` from the state election authority. `query` should be a map with :state and optional :language keys.
```
Comment (sduckett) [inline src/voter_registration_works/states.clj]: Could we wrap this line please?

File: src/voter_registration_works/states.clj [hunk 3]
```diff
@@ -71,14 +76,15 @@
 
 (defn voter-reg-authority
   [state voter]
-  (let [st-authority (state-authority state)
+  (let [st-authority (state-authority {:state state})
         reg-authority-level (-> st-authority
                                 :ocd-division
                                 (nth 2)
```
Comment (sduckett) [inline src/voter_registration_works/states.clj]: Hey, what's this `(nth 2)` doing?

**Comment by sduckett** [comment]
LGTM

File: src/voter_registration_works/states.clj (hunk 3 above)
Comment (cap10morgan) [inline src/voter_registration_works/states.clj]: It is a magic number, and one that we already have a card for obviating: https://www.pivotaltracker.com/story/show/109672026

It's a side effect of the 3-element vector representation of ocd-divisions. We need the 3rd element out of it (which is a map that holds most of what's interesting about it). Once that card is done, the whole thing should just be a map.

**Pull request merged**